app/routes/provider.py

```diff
@@ -162,8 +166,211 @@ def get_allocated_care_days(provider_id):
+        sms=f"{provider_name} invited you to join the Childcare Affordability Pilot (CAP) - access up to $1,400 monthly to pay for childcare if approved!  Click here to learn more & apply! {link} Questions? support@capcolorado.org.",
+    )
+
+
+@bp.post("/provider/invite-family")
+@auth_required(ClerkUserType.PROVIDER)
+def invite_family():
+    data = request.json
+
+    if "family_email" not in data:
+        abort(400, description="Missing required field: provider_email")
+    if "family_cell" not in data:
+        abort(400, description="Missing required field: provider_cell")
+
+    if "lang" not in data:
+        data["lang"] = "en"
+
+    user = get_current_user()
+    if user is None or user.user_data.provider_id is None:
+        abort(401)
+
+    provider_id = user.user_data.provider_id
+
+    proivder_rows = get_providers()
+
+    provider = get_provider(provider_id, proivder_rows)
+
+    if provider is None:
+        abort(404, description=f"Provider with ID {provider_id} not found.")
+
+    id = str(uuid4())
+    invitation = FamilyInvitation.new(id, data["family_email"], provider_id)
+    db.session.add(invitation)
+
+    try:
+        domain = current_app.config.get("FRONTEND_DOMAIN")
+        link = f"{domain}/invite/family/{id}"
+
+        message = get_invite_family_message(
+            data["lang"],
+            format_name(provider),
+            link,
+        )
+
+        from_email = get_from_email_internal()
+        email_sent = send_email(from_email, data["family_email"], message.subject, message.email)
+        if email_sent:
+            invitation.record_email_sent()
+
+        if data["family_cell"] is not None:
+            sms_sent = send_sms(data["family_cell"], message.sms, data["lang"])
+            if sms_sent:
+                invitation.record_sms_sent()
+    finally:
+        db.session.commit()
+
+    return jsonify({"message": "Success"}, 201)
+
+
+def get_invite_data(provider_id: int):
+    provider_rows = get_providers()
+
+    provider = get_provider(provider_id, provider_rows)
+
+    if provider is None:
+        abort(500, description=f"Provider with ID {provider_id} not found.")
+
+    return provider
+
+
+@bp.get("/provider/family-invite/<invite_id>")
+@auth_optional
+def family_invite(invite_id: str):
+    invitation_query = FamilyInvitation.invitation_by_id(invite_id)
+
+    invitation = invitation_query.first()
+
+    if invitation is None:
+        abort(404, description=f"Family invitation with ID {invite_id} not found.")
+
+    invitation.record_opened()
+    db.session.add(invitation)
+    db.session.commit()
+
+    user = get_current_user()
+
+    provider_data = get_invite_data(int(invitation.provider_google_sheet_id))
```

Comment:
Once my changes are merged this will update
```suggestion
    provider_data = get_invite_data(invitation.provider_google_sheet_id)
```